sci-electronics/kicad/kicad-5.0.0.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit check-reqs cmake-utils eapi7-ver eutils flag-o-matic gnome2-utils python-single-r1 wxwidgets vcs-snapshot xdg
+
+SERIES=$(ver_cut 1-2)
+
+DESCRIPTION="Electronic Schematic and PCB design tools."
+HOMEPAGE="http://www.kicad-pcb.org"
+SRC_URI="https://launchpad.net/${PN}/${SERIES}/${PV}/+download/${P}.tar.xz"
+
+LICENSE="GPL-2+ GPL-3+ Boost-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="doc examples github libressl +ngspice +oce +python"
+
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+"
+
+COMMON_DEPEND=">=x11-libs/wxGTK-3.0.2:${WX_GTK_VER}[X,opengl]
+	python? (
+		dev-python/wxpython:${WX_GTK_VER}[opengl,${PYTHON_USEDEP}]
+		${PYTHON_DEPS}
+	)
+	>=dev-libs/boost-1.61[context,nls,threads,python?,${PYTHON_USEDEP}]
+	github? (
+		net-misc/curl[ssl]
+		libressl? ( dev-libs/libressl:0= )
+		!libressl? ( dev-libs/openssl:0= )
+	)
+	media-libs/glew:0=
+	media-libs/freeglut
+	media-libs/mesa
+	ngspice? (
+		sci-electronics/ngspice[shared]
+	)
+	oce? (
+		sci-libs/oce
+	)
+	sys-libs/zlib
+	x11-libs/cairo"
+DEPEND="${COMMON_DEPEND}
+	doc? ( app-doc/doxygen )
+	python? ( dev-lang/swig:0 )"
+RDEPEND="${COMMON_DEPEND}
+	sci-electronics/electronics-menu
+"
+CHECKREQS_DISK_BUILD="800M"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+	setup-wxwidgets
+	check-reqs_pkg_setup
+}
+
+src_prepare() {
+	xdg_src_prepare
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DKICAD_DOCS="/usr/share/doc/${PF}"
+		-DBUILD_GITHUB_PLUGIN="$(usex github)"
+		-DKICAD_SCRIPTING="$(usex python)"
+		-DKICAD_SCRIPTING_MODULES="$(usex python)"
+		-DKICAD_SCRIPTING_WXPYTHON="$(usex python)"
+		-DKICAD_SPICE="$(usex ngspice)"
+		-DKICAD_USE_OCE="$(usex oce)"
+		-DKICAD_USE_OCC=OFF
+		-DKICAD_INSTALL_DEMOS="$(usex examples)"
+		-DCMAKE_CXX_FLAGS="-std=c++11"
+		-DCMAKE_BUILD_TYPE=Release
+	)
+	use python && mycmakeargs+=(
+		-DPYTHON_DEST="$(python_get_sitedir)"
+		-DPYTHON_EXECUTABLE="${PYTHON}"
+		-DPYTHON_INCLUDE_DIR="$(python_get_includedir)"
+		-DPYTHON_LIBRARY="$(python_get_library_path)"
+	)
+	use oce && mycmakeargs+=(
+		-DOCE_DIR='/usr/lib64'
```

Comment:
why hardcoded?

Reply:
As per documentation in KiCad this must be explicitly set. I will try to compile without this option and see what happens, but it might stay as is.

Reply:
@zpuskas it _can_ be set, yes, but not a libdir hardcoded

Reply:
@Zlogene As promised I looked into it and it's already removed as CMake will do the right thing.